### DISCUSSION TITLE
Drop Redis Dependency

### DISCUSSION THREAD

**Discussion by juliangarcess** [qruto/laravel-wave#47] [Ideas]
Will be good if we can use without redis.

**Comment by mankowitz** [comment]
Can you use the regular database driver instead?

**Comment by slavarazum** [comment]
We need a mechanism similar to [Redis's pub/sub](https://redis.io/docs/latest/develop/interact/pubsub/) to provide a central entry point for all SSE connections since the databases lack such capabilities.

I'm exploring a PHP worker solution, but scalability is a concern.
